docusaurus.config.js

```diff
@@ -146,10 +165,18 @@ const config = {
                                 label: 'Twitter',
                                 href: 'https://twitter.com/realwasmedge',
                             },
+                            {
+                                label: 'Slack#WasmEdge',
+                                to: 'https://slack.cncf.io/'
+                            },
+                            {
+                                label: 'Community Meeting',
+                                to: 'https://docs.google.com/document/d/1iFlVl7R97Lze4RDykzElJGDjjWYDlkI8Rhf8g4dQ5Rk/edit?usp=sharing'
+                            }
                         ],
                     },
                 ],
-                copyright: `Copyright © ${new Date().getFullYear()} WasmEdge. Built with Docusaurus.`,
```

Comment:
Keeping _"Built with Docusaurus"_  will be in good faith for Open Source community.

Reply:
Copyright should be small and concise.

Reply:
@Sukriti-sood Removing docusaurus is like basically going against the OSS principles. We are using a OSS framework for the site and the creators/maintainers of docusaurus project should get their credit as well. IMHO we should keep the `Built with docusaurus` message

Reply:
Sorry @adithyaakrishna @kokamkarsahil , I will revert this change. I understand the principles of OSS and not in against of giving credits to creators.